sktime/transformers/rocket.py

```diff
@@ -0,0 +1,239 @@
+from numba import njit, prange
+import numpy as np
+import pandas as pd
+
+from sklearn.utils.validation import check_is_fitted, check_random_state

```

Comment:
```suggestion
from sklearn.utils.validation import check_random_state
from sktime.utils.validation import check_is_fitted
```

Reply:
One more change as we just changed this on `dev`, should be able to simply accept the suggestions.